README.md

```diff
@@ -663,6 +726,27 @@ Calculates the amount of tax owed and the tax bands for a resident capital gains
 }
 ```
 
+## GET /capital-gains-calculator/shares/calculate-total-costs
+
+Calculates the total amount of costs in the total gain calculation for a shares capital gains user
+
+### Example of usage
+
+**Request Body**
+
+```json
+{
+    "disposalValue": 450000.0,
+    "disposalCosts": 500.0,
+    "acquisitionValue": 500000.0,
+    "acquisitionCosts": 200.0
+}
+```
+
+**Reponse**
```

Comment:
Please change to **Response**

Also there are other incorrect spellings of response. Please could you correct them.